app/Controllers/feedController.php

```diff
@@ -587,6 +588,11 @@ public static function actualizeFeeds(?int $feed_id = null, ?string $feed_url =
 								continue;
 							}
 
+							$entry->applyFilterActions($titlesAsRead);
+							if ($readWhenSameTitleInFeed > 0) {
+								$titlesAsRead[$entry->title()] = true;
+							}
+
```

Comment:
`entry_auto_unread` is called before the filter marks articles as read.
I think not calling it for filtered articles because they will be marked as read finally.

Reply:
Oh, not that part.
I think the following part calls `entry_auto_unread` before filter action.

https://github.com/FreshRSS/FreshRSS/blob/094ebf92f7c7fd9af5cb72a27198cffd23cdd4be/app/Controllers/feedController.php#L580-L583

Then apply the filter action:

https://github.com/FreshRSS/FreshRSS/blob/094ebf92f7c7fd9af5cb72a27198cffd23cdd4be/app/Controllers/feedController.php#L591

I think that the article which targeted in the filter action will remain read even though `entry_auto_unread` was called.

Reply:
Sorry, please forget my comment for once.

Hook call behavior is a different topic and seems to be consistent with existing policy.
The existing code calls `entry_auto_unread` for no read status change too (unread -> unread).

It seems unlikely that there is any need to change, because no problems have occurred now.

Reply:
Thanks for commenting again. I had not had time yet to investigate and was about to do so :-)